Firmware/RTK_Surveyor/support.ino

```diff
@@ -185,8 +185,8 @@ void systemPrintln(float value, uint8_t decimals)
 //Print a double precision floating point value with a specified number of decimal places
 void systemPrint(double value, uint8_t decimals)
 {
-  char temp[300];
```

Comment:
So I was a little paranoid... 30 is *probably* enough.

Reply:
I thought it was a typo!!??

Reply:
I may have been bitten by failing to print too many long longs that day. No idea. 30 is much more sane.